src/averageAges.js

```diff
@@ -38,6 +54,18 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const WEMEN = withChildren
+    ? people
+      .filter((person, i, persons) => person.sex === 'f'
+        && persons
```

Comment:
```suggestion
      .filter((person) => person.sex === 'f'
        && people
```
You don't need `persons`, you already have `people` parameter which is visible anywhere inside the function, you can use it as many times as you need. And definitely don't need `i`.
And the same down below.